### PR TITLE
Styles(VerticalTabsAndSelect): add hover and active color correctly and fix little bug about border radius

### PR DESCRIPTION
**Pull request by GustavoUT22**
# Fix styles about verticaltabs and select

## Summary
This PR addresses style fixes for the select component in the mobile version and adjustments to the border-top-left-radius for verticaltabs.

## Problem

Incorrect hover and active colors for the select component in mobile view.
border-top-left-radius in verticaltabs not rendering as expected on mobile devices.
## Cause
Hover and active state styles were not optimized for mobile-specific scenarios.
The border-top-left-radius worked well on larger screens but did not adapt correctly to mobile view.
## Solution
Updated the hover and active color styles for the select component to ensure proper appearance in mobile view.
Adjusted the border-top-left-radius for verticaltabs to function as expected on both desktop and mobile views.

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Refactor
- [X] Styling
- [ ] Testing
- [ ] Chore
- [ ] Other: ...

## Checklist:

These items must be completed before the code review

- [X] Check that branch is pointing to production environment
- [X] Remove console.logs
- [X] Check imports format
- [X] Check variables destructure
- [X] Check docs
- [X] Review the entire pull request yourself before sending it to the reviewer
- [X] Check task requirements on Monday or Sprint Planning sheet

### CODE REVIEW

**Review by mardecera — CHANGES_REQUESTED** [review]
image

Select options are above select.

File: src/ui/Select/Select.styles.ts [hunk 1]
```diff
@@ -27,14 +27,18 @@ export const StyledSelect = styled.div<{ $type: string }>`
     background-color: ${({ $type }) =>
       $type == 'primary'
         ? `${colors.primary.active[400]}`
-        : `${colors.secondary.active[950]}`};
+        : `${colors.tertiary.hover[300]}`};
+    color: ${({ $type }) =>
+      $type == 'primary' ? `` : `${colors.tertiary.hover[600]}`};
```
Comment (CrhistianT7) [inline src/ui/Select/Select.styles.ts:32]: use secondary type to make the ternary expression

File: src/ui/Select/Select.styles.ts [hunk 2]
```diff
@@ -27,14 +27,18 @@ export const StyledSelect = styled.div<{ $type: string }>`
...
   }
 
   &:active {
     background-color: ${({ $type }) =>
       $type == 'primary'
         ? `${colors.primary.active[600]}`
-        : `${colors.secondary.active[800]}`};
+        : `${colors.tertiary.active[400]}`};
+    color: ${({ $type }) =>
+      $type == 'primary' ? `` : `${colors.tertiary.active[700]}`};
```
Comment (CrhistianT7) [inline src/ui/Select/Select.styles.ts:41]: use secondary type to make the ternary expression

**Review by CrhistianT7 — CHANGES_REQUESTED** [review]

**Review by mardecera — APPROVED** [review]
Nice

**Pull request merged**